avalanche/training/templates/base_sgd.py

```diff
@@ -238,6 +238,14 @@ def _before_training_exp(self, **kwargs):
 
         super()._before_training_exp(**kwargs)
 
+    def _after_training(self, **kwargs):
+        super()._after_training(**kwargs)
+        # reset for faster serialization
```

Comment:
This is a good idea, but it seems that it is missing its "eval" equivalent. Also, maybe it's better to have a separate "cleanup" method called by both _after_train and _after_eval.